salt/apps/ocw/templates/engines.conf.jinja

```diff
@@ -0,0 +1,107 @@
+{% set environment = salt.grains.get('environment') %}
+
+{% if environment == 'production' %}
+{% set db_host = '10.100.0.7' %}
+{% set db_name = 'OCWDB' %}
+
+{% set cms_url = 'https://ocw-production-cms-2.odl.mit.edu' %}
+
+{% set dspace_test_host = 'sparkill.mit.edu' %}
+{% set dspace_test_endpoint = 'http://' ~ dspace_test_host ~ '/sword/deposit/' %}
+
+{% elif environment == 'qa' %}
+{% set db_host = '10.100.0.34' %}
```

Comment:
We should consider either using the Salt mine or deploying Consul to this environment. This is fine for now, but we should try to avoid statically defined IPs